### PR TITLE
refactor(example): re-init example

### LINKED ISSUE #3469: Example build failed on android
# Bug

## Platform
* Android

## Environment info
* OS: windows

Library version: 6.0.0-beta.3
Device: SAMSUNG A546E - android 14

## Steps To Reproduce

1. Clone project from github
2.  In the root directory run `yarn xbasic install` then, `yarn xbasic android`

## Expected behaviour

1. project successfully built

## Error

```
transform[stderr]: Could not resolve "E:\ReactNative\react-native-video\src\index" in file E:\ReactNative\react-native-video\examples\basic\src\VideoPlayer.tsx.
transform[stderr]: Could not resolve "E:\ReactNative\react-native-video\src\index" in file E:\ReactNative\react-native-video\examples\basic\src\MultiValueControl.tsx.
 BUNDLE  src/index.js

error: Error: Unable to resolve module @babel/runtime/helpers/interopRequireDefault from E:\ReactNative\react-native-video\src\index.ts: @babel/runtime/helpers/interopRequireDefault could not be found within the project.
> 1 | import Video from './Video';
    | ^
  2 | export {VideoDecoderProperties} from './VideoNativeComponent';
  3 | export * from './types';
  4 | export type {VideoRef} from './Video';
    at ModuleResolver.resolveDependency (E:\ReactNative\react-native-video\examples\basic\node_modules\metro\src\node-haste\DependencyGraph\ModuleResolution.js:153:15)
    at DependencyGraph.resolveDependency (E:\ReactNative\react-native-video\examples\basic\node_modules\metro\src\node-haste\DependencyGraph.js:279:43)
    at E:\ReactNative\react-native-video\examples\basic\node_modules\metro\src\lib\transformHelpers.js:176:21
    at resolveDependencies (E:\ReactNative\react-native-video\examples\basic\node_modules\metro\src\DeltaBundler\buildSubgraph.js:56:25)
    at visit (E:\ReactNative\react-native-video\examples\basic\node_modules\metro\src\DeltaBundler\buildSubgraph.js:107:30)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Promise.all (index 8)
    at async visit (E:\ReactNative\react-native-video\examples\basic\node_modules\metro\src\DeltaBundler\buildSubgraph.js:116:5)
    at async Promise.all (index 2)
    at async visit (E:\ReactNative\react-native-video\examples\basic\node_modules\metro\src\DeltaBundler\buildSubgraph.js:116:5)
```

### PR DESCRIPTION
**Pull request by KrzysztofMoch**
## Summary
To clean code and catch up with newest template I have re-created basic example

**NOTE**: I remove windows from basic example because it's very heavy - should (will) be moved to separate example

fixes #3469

## Changes (PR change log is little mess... 😅)
- re-inited `basic` example
- removed `FabricExample` & `embed-and-fullscreen` examples (`basic` should be used to test fabric)

## Test Plan
- [x] App build and start with:
  1. `yarn install`
  2. `yarn build`
  3. `yarn xbasic install`
  4. `yarn xbasic android`
- [ ] CI pass

### CODE REVIEW

**Comment by freeboub** [comment]
@KrzysztofMoch did you see the check didn't pass ?

**Comment by KrzysztofMoch** [comment]
Yes, did not have time to fix it 🥲 (I forgot to move lines from old podfile)

**Comment by freeboub** [comment]
@KrzysztofMoch When building from scratch I still have the build issue:

It is really not clear why I have this issue when cloning from scratch, but not on my previously cloned repo :/ 

transform[stderr]: Could not resolve "/Users/olivier/Documents/GitHub/react-native-video/src/index" in file /Users/olivier/Documents/GitHub/react-native-video/examples/basic/src/VideoPlayer.tsx.
transform[stderr]: Could not resolve "/Users/olivier/Documents/GitHub/react-native-video/src/index" in file /Users/olivier/Documents/GitHub/react-native-video/examples/basic/src/MultiValueControl.tsx.
 BUNDLE  src/index.js 

error: Error: Unable to resolve module @babel/runtime/helpers/interopRequireDefault from /Users/olivier/Documents/GitHub/react-native-video/src/index.ts: @babel/runtime/helpers/interopRequireDefault could not be found within the project or in these directories:


Note: I also think you broken windows sample... it will need a reallignment in the future

File: examples/basic/android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,23 +1,13 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
 
-    <uses-feature
```
Comment (freeboub) [inline examples/basic/android/app/src/main/AndroidManifest.xml:3]: can you please revert changes in this file to keep "current" android tv support ?

File: examples/basic/package.json [hunk 2]
```diff
@@ -1,41 +1,34 @@
 {
   "name": "VideoPlayer",
-  "version": "1.0.0",
+  "version": "0.0.1",
   "private": true,
   "scripts": {
     "android": "react-native run-android",
     "ios": "react-native run-ios",
-    "windows": "react-native run-windows",
```
Comment (freeboub) [inline examples/basic/package.json:8]: keep windows please

File: examples/basic/package.json [hunk 3]
```diff
@@ -1,41 +1,34 @@
...
+    "lint": "eslint .",
     "start": "react-native start",
-    "test": "jest",
-    "lint": "eslint ."
+    "test": "jest"
   },
   "dependencies": {
-    "@react-native-picker/picker": "^1.9.11",
+    "@react-native-picker/picker": "react-native-picker/picker",
```
Comment (freeboub) [inline examples/basic/package.json]: strange, what does it do ?

File: examples/basic/package.json [hunk 4]
```diff
@@ -1,41 +1,34 @@
...
     "react": "18.2.0",
-    "react-native": "0.73.2",
-    "react-native-windows": "0.63.41"
```
Comment (freeboub) [inline examples/basic/package.json:17]: keep react-native-windows please

**Review by freeboub — CHANGES_REQUESTED** [review]

File: examples/basic/package.json (hunk 3 above)
Comment (KrzysztofMoch) [reply examples/basic/package.json]: I really don't know 😅 i will remove it ...

**Comment by KrzysztofMoch** [comment]
I will close this one as there is a lot of conflicts and current examples seems to work again

**Pull request closed without merging**